src/server/conf/passport.js

```diff
@@ -1,5 +1,9 @@
+const bcrypt = require('bcrypt');
+
 const appConf = require('./server');
 
+const salt = bcrypt.genSalt(process.env.SALT_ROUNDS || 10); // It's a `Promise<__SALT__>`
```

Comment:
I think its better to use:
```
bcrypt.hash(myPlaintextPassword, saltRounds, function(err, hash) {
  // Store hash in your password DB.
});
```
I guess in this case we'll have different hashed for different users